src/DssSpell.t.sol

```diff
@@ -114,6 +114,40 @@ contract DssSpellTest is DssSpellTestBase {
     }
     // END OF TESTS THAT SHOULD BE RUN ON EVERY SPELL
 
+    struct DebtCeilingParams {
+        uint256 line;
+        uint256 debt;
+    }
+
+    // Avoids issues with stack too deep in `testGlobalDebtCeilingDecrease`
+    function _getDebtCeilingParams(bytes32 ilk) internal view returns (DebtCeilingParams memory) {
+        (uint256 Art, uint256 rate, , uint256 line, ) = vat.ilks(ilk);
+
+        return DebtCeilingParams({
+            line: line,
+            debt: Art * rate
+        });
+    }
+
+    function testGlobalDebtCeilingDecrease() public {
```

Comment:
There is nothing functionally wrong with this, but I want to point out:

- A novel struct has been used
- Variable names don't match how we do varnames
- There are ternary operators in this which make it less readable

New tests should go at the end of the spell tests rather than the start. 
There are likely examples of how this was done in another way in the archive.

Reply:
Regarding the struct in the test, that's a way around stack size limitation.

Another way of doing it would be mirroring the code in the spell, but that's exactly what you DON'T want to do in a test if you can, otherwise you are not really testing anything.

Reply:
I'm agree with you on this and the code has been cleaned up. Resolving issue.